full_build.sh

```diff
@@ -33,6 +33,8 @@ sh build/build-rector-scoped.sh rector-build rector-prefixed-downgraded
 cd rector-prefixed-downgraded
 cp ../build/target-repository/bootstrap.php .
 cp ../preload.php .
-bin/rector list --ansi && cd ..
+bin/rector list --ansi
+bin/rector process vendor/symfony/string/Slugger/ --dry-run
```

Comment:
This step should not be needed, the downgrade should happen in one run. Why it's added?

Reply:
The full_build.sh is a local script, never run in CI, just to verify that run against Slugger is working ok

Reply:
I see :+1:  thanks for explaning, I thought this is workflow job